src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/utils/ManageCaseHeader.scala

```diff
@@ -33,4 +33,24 @@ object ManageCaseHeader {
     "Connection" -> "keep-alive"
   )
 
+  /*NOC -  Get Questions Header*/
+  val nocGetQuestionsHeader = Map(
+    "ServiceAuthorization" -> "Bearer ${ccd_dataBearerToken}",
+    "Authorization" -> "Bearer ${access_token}",
+    "Accept" -> "*/*",
+    "Host" -> manageCaseHost,
+    "Accept-Encoding" -> "gzip, deflate, br",
+    "Connection" -> "keep-alive"
+  )
+
+  /*NOC -  Post Questions Header*/
+  val nocPostQuestionsHeader = Map(
```

Comment:
This looks like the same as the above one? Just have one header and rename to something like `nocQuestionsHeader` ?

Reply:
header file has been refined to 1 aac header across the aac requests with update to the PR